### ISSUE TITLE
Merge development branch ("non-open") for journal-scrapers

### ISSUE THREAD

**Issue by petermr** [ContentMine/journal-scrapers#35]
There is a "non-open" branch with some scrapers for journals not on the main branch, but > 1 year old. What is our philosophy? I assume "non-open" actually means "development", not "non-open_access" and should be renamed.
It's important we expose our development clearly as elsewise people will repeat development. Since many people don't understand Git branches, I suggest a `development` or `sandbox` directory. That will encourage new people to participate.

**Comment by robintw** [comment]
I agree with this, it was only thanks to @rossmounce that I found out about the non-open branch.
